src/apps/api/views/competitions.py

```diff
@@ -236,6 +236,20 @@ class PhaseViewSet(ModelViewSet):
     serializer_class = PhaseSerializer
     # TODO! Security, who can access/delete/etc this?
 
+    @action(detail=True, methods=('POST',))
+    def manually_migrate(self, request, pk):
+        """
+        Manually migrates _from_ this phase. The destination phase does not need auto migration set to True
+        """
+        phase = self.get_object()
```

Comment:
on a detail action view this isn't done already? I looked through a bit and didn't see this done already in some inherited class... so it may need to be done?